frontend/src/api/useAccessReview.ts

```diff
@@ -29,17 +29,29 @@ const checkAccess = ({
       },
     },
   };
-  return k8sCreateResource<SelfSubjectAccessReviewKind>({
-    model: SelfSubjectAccessReviewModel,
-    resource: selfSubjectAccessReview,
-  });
+  return (
+    k8sCreateResource<SelfSubjectAccessReviewKind>({
+      model: SelfSubjectAccessReviewModel,
+      resource: selfSubjectAccessReview,
+    })
+      // TODO: TypeScript doesn't realize this can be inferred as this type and thinks it is boolean[]
+      .then((result): [allowed: boolean, loaded: boolean] => [
+        result.status ? result.status.allowed : true,
+        true,
+      ])
+      .catch((e) => {
+        // eslint-disable-next-line no-console
+        console.warn('SelfSubjectAccessReview failed', e);
+        return [true, true]; // if it critically fails, don't block SSAR checks; let it fail/succeed on future calls
+      })
+  );
 };
```

Comment:
loaded is never not true. is that intended

Reply:
SSAR cannot be assumed to have failed -- if we get it wrong, the UI never works on that cluster (theoretically) -- this is not security... this is conveienence. K8s still validates them on submit.

Reply:
regardless i dont think it can ever be not loaded so whats the point of having that value